testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/audit/AddAggregateSecurityEventListenerOnlineTest.java

```diff
@@ -0,0 +1,209 @@
+        client.apply(addAggregateSecurityEventListener2);
+
+        assertTrue("Aggregate security event listener should be created",
+                ops.exists(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS));
+        assertTrue("Second aggregate security event listener should be created",
+                ops.exists(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS));
+    }
+
+    @Test(expected = CommandFailedException.class)
+    public void addExistAggregateSecurityEventListenersNotAllowed() throws Exception {
+        AddAggregateSecurityEventListener addAggregateSecurityEventListener
+                = new AddAggregateSecurityEventListener.Builder(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_NAME)
+                .addSecurityEventListeners(TEST_FILE_AUDIT_LOG_NAME, TEST_SYSLOG_AUDIT_LOG_NAME)
+                .build();
+        AddAggregateSecurityEventListener addAggregateSecurityEventListener2
+                = new AddAggregateSecurityEventListener.Builder(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_NAME)
+                .addSecurityEventListeners(TEST_FILE_AUDIT_LOG_NAME, TEST_SYSLOG_AUDIT_LOG_NAME)
+                .build();
+
+        client.apply(addAggregateSecurityEventListener);
+        assertTrue("Aggregate security event listener should be created",
+                ops.exists(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS));
+        client.apply(addAggregateSecurityEventListener2);
+        fail("Aggregate security event listener CreaperTestAggregateSecurityEventListener already exists in configuration, exception should be thrown");
+    }
+
+    @Test(expected = CommandFailedException.class)
+    public void addExistAggregateSecurityEventListenersAllowed() throws Exception {
+        AddAggregateSecurityEventListener addAggregateSecurityEventListener
+                = new AddAggregateSecurityEventListener.Builder(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_NAME)
+                .addSecurityEventListeners(TEST_FILE_AUDIT_LOG_NAME, TEST_SYSLOG_AUDIT_LOG_NAME)
+                .build();
+        AddAggregateSecurityEventListener addAggregateSecurityEventListener2
+                = new AddAggregateSecurityEventListener.Builder(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_NAME)
+                .addSecurityEventListeners(TEST_FILE_AUDIT_LOG_NAME, TEST_FILE_AUDIT_LOG_NAME2)
+                .build();
+
+        client.apply(addAggregateSecurityEventListener);
+        assertTrue("Aggregate security event listener should be created",
+                ops.exists(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS));
+        client.apply(addAggregateSecurityEventListener2);
+
+        // check whether it was really rewritten
+        checkAttribute(TEST_AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS, "security-event-listeners[1]",
+                TEST_FILE_AUDIT_LOG_NAME2);
+    }
+
+    @Test(expected = CommandFailedException.class)
+    public void addAggregateSecurityEventListener_notConfiguredSecurityEventListener() throws Exception {
```

Comment:
This does not not test Creaper command. It tests model, I suggest to move it to different testsuite.